includes/wikia/models/DesignSystemSharedLinks.class.php

```diff
@@ -199,10 +194,9 @@ private function getLangWithFallback( $lang ) {
 			'user-register' => 'https://www.wikia.com/register?uselang=pl',
 		],
 		'pt-br' => [
-			'explore-wikis' => 'http://fandom.wikia.com/explore?uselang=pt-br',
-			'fandom-logo' => 'http://pt-br.wikia.com',
+			'explore-wikis' => 'http://pt-br.wikia.com/wiki/Wikia_em_Portugu%C3%AAs',
```

Comment:
Do we need the `/wiki/Wikia_em_Portugu%C3%AAs` part of the URL? `http://pt-br.wikia.com` redirects to the main page, which's name can change—which would break it.